iica-catie.csl

```diff
@@ -234,118 +228,147 @@
     </sort>
     <layout>
       <choose>
-        <if type="bill book" match="any">
+        <if type="book bill article-journal" match="any">
           <group delimiter=". ">
             <text macro="author-short"/>
             <text macro="year-date"/>
             <text macro="title"/>
+            <choose>
+              <if match="any" variable="URL">
+                <text term="online" prefix="(" suffix=")"/>
+              </if>
+            </choose>
             <text macro="translator"/>
             <text macro="editor"/>
```

Comment:
@danirv91, the style also doesn't validate right now. One reason is that you call the "editor" macro twice (here and somewhere further down), but you removed this macro.

Reply:
The other issue is that the style defines a number of macros that aren't used, which we don't allow for repository styles (unused macros should just be deleted). These macros are: "Editor", "container-author", "date", "issue".

Reply:
Hello, if those macros are not being used, so they can be deleted. Regarding the macro "editor", I'm afraid of removing that calls, I don't know If that could affect the style. Sorry I don't know much about this language, I developped the style with the visual editor, and I only saw that it works good.

Reply:
@POBrien333, do you have time to look this over? (you could also restart from the current repository version and check the new version of the style guide, if that's any easier)

If not, let me know, and I'll fix up the current submission.